src/test/steps/loginSteps.ts

```diff
@@ -1,37 +1,42 @@
 import {Given, When, Then} from '@cucumber/cucumber';
-import {chromium, Page, Browser, expect} from '@playwright/test';
-
-let browser: Browser;
-let page: Page;
-
-Given('User navigates to the application', async function () {
-  browser = await chromium.launch({headless: false});
-  page = await browser.newPage();
-  page.goto('https://px-stage.self-service.danads.com');
+import {pageFixture} from '../../hooks/pageFixture';
+import Assert from '../../helper/wrapper/assert';
+import AuthPage from '../../pages/basicAuthPage';
+import LoginPage from '../../pages/loginPage';
+
+let authPage: AuthPage;
+let loginPage: LoginPage;
+let assert: Assert;
```

Comment:
Are you going to create this object for every page step definition?

Reply:
Yes

Reply:
fixed